server/odc-service/src/main/java/com/oceanbase/odc/service/datasecurity/model/SensitiveColumnScanningTaskInfo.java

```diff
@@ -43,8 +42,8 @@ public class SensitiveColumnScanningTaskInfo {
     private ErrorCode errorCode;
     private String errorMsg;
 
-    public SensitiveColumnScanningTaskInfo(@NonNull Long projectId, @NonNull Integer allTableCount) {
-        this.taskId = StringUtils.uuid();
+    public SensitiveColumnScanningTaskInfo(String taskId, @NonNull Long projectId, @NonNull Integer allTableCount) {
```

Comment:
Do we need a `NotNull` annotation here

Reply:
Of course, adding @NonNull is better, but since @Validated is not added, this annotation is only used by the compiler and does not actually perform validation.